### PR TITLE
XWPFParagraph: direct control over keepNext.

### PR DESCRIPTION
**Pull request by dourouc05**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFParagraph.java [hunk 1]
```diff
@@ -402,6 +402,24 @@ public BigInteger getNumStartOverride() {
         }
         return null;
     }
+    
+    /**
+     * Indicates whether this paragraph should be kept on the same page as the next one.
+     */
+    public boolean isKeepNext() {
+        if (getCTP() != null && getCTP().getPPr() != null) {
```
Comment (pjfanning) [inline src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFParagraph.java]: does this compile? seems to me, this has no return if the if-condition fails

Comment (dourouc05) [reply src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFParagraph.java]: I've corrected this and added a test. Better?

**Comment by pjfanning** [comment]
this is merged - https://github.com/apache/poi/pull/156 had the same code and I merged it there

**Pull request closed without merging**